tools/ci_build/github/azure-pipelines/templates/rocm.yml

```diff
@@ -73,7 +72,7 @@ jobs:
 
   # All UTs were here are now covered in AMD CI - see orttraining-pai-ci-pipeline.yml
```

Comment:
>orttraining-pai-ci-pipeline.yml [](http://example.com/codeflow?start=54&length=31)

should we move amd CI to use 5.2.3 whl and enable ortmodule test once? Or we can move/test after 5.3, thought?

Reply:
Yes, we should keep amd CI to the latest rocm version we use. I will create a another pr to do this.